google/api/control/path_template.py

```diff
@@ -188,15 +188,14 @@ def parse(self, data):
         # Validation step: checks that there are no nested bindings.
         path_wildcard = False
         for segment in segments:
-            if segment.kind == _TERMINAL and segment.literal == '**':
+            if segment.kind == _TERMINAL and segment.literal == u'**':
                 if path_wildcard:
                     raise ValidationException(
-                        'validation error: path template cannot contain more '
-                        'than one path wildcard')
+                        u'validation error: path template cannot contain more 'u'than one path wildcard')
```

Comment:
Yeah, my unicodification script does this by accident and I didn't catch this incident.